docs/examples/overlaypanel/confirmationExample.js

```diff
@@ -0,0 +1,193 @@
+// @flow strict
+import React, { type Node, useCallback } from 'react';
+import {
+  Box,
+  Button,
+  CompositeZIndex,
+  FixedZIndex,
+  Flex,
+  Layer,
+  OverlayPanel,
+  Text,
+  TextField,
+  Fieldset,
+  Checkbox,
+  RadioButton,
+} from 'gestalt';
+
+export default function Example(): Node {
+  const [shouldShow, setShouldShow] = React.useState(false);
+  const HEADER_ZINDEX = new FixedZIndex(10);
+  const sheetZIndex = new CompositeZIndex([HEADER_ZINDEX]);
+
+  const renderFooter = useCallback(
+    ({ onDismissStart }) => (
+      <Flex alignItems="center" justifyContent="end">
+        <Button color="red" text="Create" onClick={onDismissStart} />
+      </Flex>
+    ),
+    [],
+  );
+
+  return (
+    <React.Fragment>
```

Comment:
import this directly, don't import `React`

Reply:
DONE